manifests/config.pp

```diff
@@ -61,16 +61,29 @@
     notify  => Class['jira::service'],
   }
 
-  file { "${jira::webappdir}/conf/server.xml":
-    content => template('jira/server.xml.erb'),
-    mode    => '0600',
-    require => Class['jira::install'],
-    notify  => Class['jira::service'],
-  }->
+  if ($version[0] == '7' ){
+    file { "${jira::webappdir}/conf/server.xml":
+      content => template('jira/server7.xml.erb'),
+      mode    => '0600',
+      require => Class['jira::install'],
+      notify  => Class['jira::service'],
+    }
+  }else{
+    file { "${jira::webappdir}/conf/server.xml":
+      content => template('jira/server.xml.erb'),
```

Comment:
I'd prefer we replaced the entire xml file template with augeas. That way we can keep up to date with all the latest modifications to the server.xml when we upgrade but still customize it.